pinot-plugins/pinot-input-format/pinot-protobuf/src/test/java/org/apache/pinot/plugin/inputformat/protobuf/ComplexTypes.java

```diff
@@ -719,8 +719,12 @@ public int getNestedIntField() {
       @java.lang.Override
       public final boolean isInitialized() {
         byte isInitialized = memoizedIsInitialized;
-        if (isInitialized == 1) return true;
-        if (isInitialized == 0) return false;
+        if (isInitialized == 1) {
+          return true;
+        }
+        if (isInitialized == 0) {
+          return false;
+        }
```

Comment:
Please avoid unrelated formatting changes specially in a large PR like this one. I would suggest reverting all changes that are strictly not part of this PR.

Reply:
Yeah, I think my global format script just fixed all checkstyle violations :). Reverted, thanks